### PR TITLE
163 use tts to speak questions

### PR DESCRIPTION
**Pull request by Guekka**

### CODE REVIEW

File: apps/backend/api/src/schemas/tts.schema.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { boolean, object, string, TypeOf } from 'zod';
+
+export const ttsSchema = object({
```
Comment (Apoorva64) [inline apps/backend/api/src/schemas/tts.schema.ts:3]: shemas should be in shared types lib

File: apps/backend/api/src/controllers/tts.controller.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      .getAllAudioBase64(text, {
+        lang: 'fr',
+        slow: slow,
+        host: 'https://translate.google.com',
+        timeout: 10000,
+        splitPunct: ',.?',
+      })
+      .then((audios) => audios.flatMap((audio) => audio.base64).join(''));
+
+    res.send({ message: audio, text: text });
```
Comment (Apoorva64) [inline apps/backend/api/src/controllers/tts.controller.ts:24]: create a type for this and wrap it with RequestWrapper

**Comment by Apoorva64** [comment]
# ☁️ Nx Cloud Report

CI is running/has finished running commands for commit e045378aaad89f4586626eeecd9f1732321ffa57. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=640ee8fb56da4d6500e89717&branchName=200,163-use-tts-to-speak-questions)

---


<details>
<summary>✅ Successfully ran 3 targets </summary>

- [`nx serve backend-api`](https://cloud.nx.app/runs/gG4BeRhmAP)
- [`nx affected --target=test --parallel=3 --configuration=ci`](https://cloud.nx.app/runs/ddp8agM4Av)
- [`nx affected --target=lint --parallel=3`](https://cloud.nx.app/runs/soj7wsx6Jf)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

File: apps/front-end/front-office/project.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "prefix": "webonjour",
   "targets": {
     "build": {
-      "executor": "@angular-devkit/build-angular:browser",
+      "executor": "@angular-builders/custom-webpack:browser",
```
Comment (Apoorva64) [inline apps/front-end/front-office/project.json:9]: rollback this

File: libs/front-end/shared/common/src/lib/services/tts/tts.service.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+export class TtsService {
+  API_URL = 'http://localhost:8000';
+  private audio = new Audio();
+
+  constructor(private http: HttpClient) {}
+
+  sayTTS(text: string, slow: boolean = false) {
+    this.http
+      .get<RequestWrapper<Tts.TtsResponse>>(
+        `${this.API_URL}/api/tts?text=${text}&slow=${slow}`
```
Comment (ozeliurs) [inline libs/front-end/shared/common/src/lib/services/tts/tts.service.ts:20]: Slow parameter not used

**Pull request merged**